<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWG</class>
 <widget class="QWidget" name="ExportWG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="save_dir_le">
        <property name="placeholderText">
         <string>Input Save Dir</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="save_filename_le">
        <property name="placeholderText">
         <string>Input Save File Name</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="preview_btn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Preview</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="export_btn">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="export_fileds_gbox">
     <property name="title">
      <string>Export Fileds</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="compact_cbox">
        <property name="text">
         <string>compact(csv)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>item_sep:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="compact_item_sep_le">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>,</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="compact_nokey_cbox">
          <property name="text">
           <string>nokey</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>escape:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="compact_escape_combox">
          <item>
           <property name="text">
            <string>c</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>csv</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="compact_print_section_cbox">
          <property name="text">
           <string>print_section</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>suffix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="compact_suffix_le">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>.csv</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="ini_cbox">
        <property name="text">
         <string>ini</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="default_nokey_cbox">
          <property name="text">
           <string>nokey</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="default_noprint_wrappers_cbox">
          <property name="text">
           <string>noprint_wrappers</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>suffix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="default_suffix_le">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>.txt</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="default_cbox">
        <property name="text">
         <string>default</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="flat_hierarchical_cbox">
          <property name="text">
           <string>hierarchical</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>sep_char:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="flat_sep_char_le">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>.</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>suffix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="flat_suffix_le">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>.txt</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="ini_hierarchical_cbox">
          <property name="text">
           <string>hierarchical</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>suffix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ini_suffix_le">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>.ini</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="json_cbox">
        <property name="text">
         <string>json</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="json_compact_cbox">
          <property name="text">
           <string>compact</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>suffix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="json_suffix_le">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>.json</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="flat_cbox">
        <property name="text">
         <string>flat</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="xml_fully_qualified_cbox">
          <property name="text">
           <string>fully_qualified</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="xml_xsd_strict_cbox">
          <property name="text">
           <string>xsd_strict</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>suffix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="xml_suffix_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>.xml</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="xml_cbox">
        <property name="text">
         <string>xml</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>default_cbox</tabstop>
  <tabstop>default_nokey_cbox</tabstop>
  <tabstop>default_noprint_wrappers_cbox</tabstop>
  <tabstop>default_suffix_le</tabstop>
  <tabstop>json_cbox</tabstop>
  <tabstop>json_compact_cbox</tabstop>
  <tabstop>json_suffix_le</tabstop>
  <tabstop>ini_cbox</tabstop>
  <tabstop>ini_hierarchical_cbox</tabstop>
  <tabstop>ini_suffix_le</tabstop>
  <tabstop>flat_cbox</tabstop>
  <tabstop>flat_hierarchical_cbox</tabstop>
  <tabstop>flat_sep_char_le</tabstop>
  <tabstop>flat_suffix_le</tabstop>
  <tabstop>compact_cbox</tabstop>
  <tabstop>compact_item_sep_le</tabstop>
  <tabstop>compact_nokey_cbox</tabstop>
  <tabstop>compact_escape_combox</tabstop>
  <tabstop>compact_print_section_cbox</tabstop>
  <tabstop>compact_suffix_le</tabstop>
  <tabstop>xml_cbox</tabstop>
  <tabstop>xml_fully_qualified_cbox</tabstop>
  <tabstop>xml_xsd_strict_cbox</tabstop>
  <tabstop>xml_suffix_le</tabstop>
  <tabstop>save_dir_le</tabstop>
  <tabstop>save_filename_le</tabstop>
  <tabstop>preview_btn</tabstop>
  <tabstop>export_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
